### DISCUSSION TITLE
Rotate specific plane to x-axis of a pole figure

### DISCUSSION THREAD

**Discussion by lebraunjames** [mtex-toolbox/mtex#1913] [Ask Anything]
Hello MTEX community, 

I have a question about plotting in pole figures. My aim is to align the [1,-1,0] plane of parentOri with the x-axis in pole figure. However, I am struggling to figure out how to correctly rotate my data using MTEX to achieve this.

Here's the code I've been working with:
```
cs_aus = ebsd('Iron fcc').CS;
parentOri = orientation.byEuler(319*degree,34*degree,79*degree, cs_aus);

% I'm not sure what should go here to achieve the desired rotation
rot = rotation.byAxisAngle( ?? );
ebsd_rot = rotate(ebsd, rot, 'keepXY');

h_fcc = Miller({1,0,0}, cs_aus);
plot(parentOri * h_fcc(1).symmetrise, 'MarkerFaceColor', 'r')
```
Could you please guide me on how I can appropriately rotate the data such that the [1,-1,0] direction aligns with the x-axis?

Your assistance is greatly appreciated.

Best regards,
Jubert

**Comment by kilir** [comment]
Hi,
how about that:
```
cs = crystalSymmetry('m-3m')
parentOri = orientation.byEuler(319*degree,34*degree,79*degree, cs);
h = Miller(1,-1,0, cs);

plotPDF(parentOri,h)
hold on

% rotate parentOri such that 1,-1,0 is parallel to vector3d(0,1,0)
r = parentOri * h

% one potential rotation axis
ax = cross(r,vector3d(0,1,0));

% rotation angle
rang = angle(r,vector3d(0,1,0));

% rotate parentOri
rot = rotation.byAxisAngle(ax,rang);
oriRot = rot * parentOri

plotPDF(oriRot,h)
annotate(yvector,'labeled')
hold off
```
Cheers,
Rüdiger

**Reply by lebraunjames** [reply]
You're awesome, this works! I truly appreciate your help. 

Just a minor problem, the label shows "y" instead of the actual plane in that axis. Is there a setting that i need to tweak?

![ylabel](https://github.com/mtex-toolbox/mtex/assets/133045135/fa7ffd62-4fa8-4458-b873-8097ce44dea5)
